<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>caesar</class>
 <widget class="QMainWindow" name="caesar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Шифр Цезаря</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="inLabel">
       <property name="text">
        <string>Выберите входной файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Содержимое</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="outLabel">
       <property name="text">
        <string>Выберите выходной файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Содержимое</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="keyLabel">
       <property name="text">
        <string>Введите Ключ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>161</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="openInput">
       <property name="text">
        <string>Открыть..</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dataInput"/>
     </item>
     <item>
      <widget class="QPushButton" name="openOutput">
       <property name="text">
        <string>Открыть..</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dataOutput"/>
     </item>
     <item>
      <widget class="QSpinBox" name="keyBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>351</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="encodeButton">
       <property name="text">
        <string>Кодировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="decodeButton">
       <property name="text">
        <string>Декодировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>encodeButton</zorder>
   <zorder>decodeButton</zorder>
   <zorder>keyEdit</zorder>
   <zorder>inLabel</zorder>
   <zorder>outLabel</zorder>
   <zorder>keyLabel</zorder>
   <zorder>dataInput</zorder>
   <zorder>dataOutput</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>openInput</zorder>
   <zorder>openOutput</zorder>
   <zorder>decodeButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>файл</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action">
   <property name="text">
    <string>открыть файл</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
